tests/src/test/scala/cats/tests/EitherTTests.scala

```diff
@@ -46,17 +47,24 @@ class EitherTTests extends CatsSuite {
 
   {
     //if a Monad is defined
+    import cats.instances.option
```

Comment:
I wonder why this line is needed. I think `CatsSuite` should provide all the instance. What happens if you remove this line?